libmscore/fret.cpp

```diff
@@ -1346,6 +1346,13 @@ QVariant FretDiagram::propertyDefault(Pid pid) const
       return Element::propertyDefault(pid);
       }
 
+void FretDiagram::endEditDrag(EditData& editData)
+    {
+    Element::endEditDrag(editData);
+
+    triggerLayout();
+    }
+
```

Comment:
Please add function name comment, fix the indentation and delete that empty space.